media-libs/vigra/vigra-1.12.3.recipe

```diff
@@ -72,17 +70,19 @@ BUILD_REQUIRES="
 	devel:libz$secondaryArchSuffix
 	"
 BUILD_PREREQUIRES="
+	haiku${secondaryArchSuffix}_devel
 	cmd:cmake
 	cmd:g++$secondaryArchSuffix
 	cmd:ld$secondaryArchSuffix
 	cmd:make
-	cmd:python # vigra-config is python script
 	"
 
 PATCH()
 {
 	sed -i 's,\/lib,/'${relativeLibDir}',g' \
 		config/vigra-config.in
+	sed -e 's,\#!/usr/bin/env python,\#!/usr/bin/env python3,g' -i \
```

Comment:
```suggestion
	sed -e 's,#!/usr/bin/env python,#!/usr/bin/env python3,' -i \
```

Should work, I think. Removed the "g" because there's only one place to do the replace.